CGPs/cgp-0053.md

```diff
@@ -0,0 +1,100 @@
+
+- Running validators: 110
+- Elected validators: 110
+- Proxied validators: 10
+- Block time: 5 seconds
+- Epoch: Each 500 blocks
+- Celo-blockchain version: 1.5.5
+- Celo-monorepo (protocol) version: f62781066e5534d591c65bbcea8592cfb90d3a56
+- Validator resources: CPU = 8 vCPUs (n2 GCP platform); Memory = 32GiB
+- Proxy resources: CPU = 4 vCPUs (n2 GCP platform); Memory = 16GiB
+- Block Gas Limit illustrated in these results: 50 million
+- The network was running on "preemptible" instances that can eventually be restarted by the cloud provider (this causes some outsiders in the graphs caused by pod restarts)
+
+The load transactions were generated as token transfers, transferring different tokens and also using different currencies for paying the transaction fees. The network were running with a continued 50M load for 24 hours in different intervals.
+
+![Block Gas Used](https://user-images.githubusercontent.com/5635989/167686033-5ac16aa4-254b-446e-a90c-c4dabdb77bc2.png)
+
+As summary for the results, we illustrate some metrics for a 110 validator network, with 50M block size, running on GCP n2-standard-8 (8vCPU and 32 GiB memory).
+
+![Block Processing Time](https://user-images.githubusercontent.com/5635989/167684677-a8758d16-9998-4c11-98a7-9cf5dabdef37.png)
+
+The previous image represents how the block time (~5 seconds), split between the different steps of block construction and block consensus. The time spent in `consensus_istanbul_backend_sleep`, that represents idle time for the block time.
+
+For a 50M block size, a full node running on n2-standard-4 (4vCPU and 16 GiB memory) can keep synced without problems:
+
+![Full nodes block delta](https://user-images.githubusercontent.com/5635989/167696090-25f266ab-a4b1-48e9-82a5-dc7f95ffb4cf.png)
+
+In the previous graph is represented the block delta between the network block head, and the latest block from each node. As show, none of the instances distanced more than one blocks from the head, and part of this can be caused by different timing with the metric probe (the spike shown in the graph was caused to a restart of the preemptible instance).
+
+![Block Rate](https://user-images.githubusercontent.com/5635989/167696586-e0a0c964-8bff-49cc-a762-6cc9b8ae705f.png)
+
+Lastly, this image shows the per-second average of new blocks.
+
+In the view of the results showing stability in all the observed metrics, we can reach the conclusion that the network can handle an increment on the gas block limit to 50M without notoriously impacting the average block time.
+
+As a precaution measure, taking in count that the stress tests performed are not fully representative of all the possible usages of the network, we are also suggesting to change the gasPriceMinimum `target density` to `0.7`, to discourage long periods of stress on the network, making virtually impossible to fill more than 10 minutes of full blocks due to the high costs it would require.
```

Comment:
```suggestion
As a precautionary measure, taking into account that the stress tests performed are not fully representative of all the possible usages of the network, we are also suggesting to change the gasPriceMinimum `target density` to `0.7`, to discourage long periods of stress on the network, making virtually impossible to fill more than 10 minutes of full blocks due to the high costs it would require.
```